psclient/src/main/java/com/majeur/psclient/service/observer/BattleRoomMessageObserver.kt

```diff
@@ -867,5 +905,26 @@ class BattleRoomMessageObserver(service: ShowdownService)
         fun onSideChanged(player: Player, side: String, start: Boolean)
         fun onVolatileStatusChanged(id: PokemonId, vStatus: String, start: Boolean)
         fun onPrintBattleMessage(message: CharSequence)
+        fun onSetBattleType(type: BattleType)
+        fun goToLatest()
+        fun onEndOfReplay()
```

Comment:
We should use the regular onBattleEnded() callback.

Reply:
Done!